Splitter: add demo description (#27094)
DevExpress/DevExtreme: apps/demos/Demos/Splitter/Overview/description.md

> Use Splitter widgets to create resizable panes on a page.

Reviewer: ```suggestion
You can use Splitter widgets to create resizable panes on a page.
```

> Splitter is a component designed to split a page layout into multiple panes.

Reviewer 2: I usually try to avoid "textbook-style" definitions (like when you say "Splitter is a component that ..."). I change them into something imperative (such as "Use a Splitter to ...").

```suggestion
Use Splitter widgets to create resizable panes on a page.
```

[note] To get started with DevExtreme Splitter, refer to the following tutorial for step-by-step instructions: [Getting Started with Splitter](/Documentation/Guide/UI_Components/Splitter/Getting_Started_with_Splitter/).

> Splitter panes can be specified in the [items](/Documentation/ApiReference/UI_Components/dxSplitter/Configuration/#items) or [dataSource](/Documentation/ApiReference/UI_Components/dxSplitter/Configuration/#dataSource) properties. Use **dataSource** if data is remote or should be processed.

Reviewer 2: I'm not sure how this works. Both `items` and `dataSource` imply any number of elements. But I guess that should be limited to 2 in case of splitters? Or if you specify more than 2 then multiple splitters can appear one after another? 

```suggestion
To specify splitter panes, set the [items](/Documentation/ApiReference/UI_Components/dxSplitter/Configuration/#items) or [dataSource](/Documentation/ApiReference/UI_Components/dxSplitter/Configuration/#dataSource) property. Use **dataSource** if data is remote or should be processed. If you specify multiple panes, they will appear one after another with splitters between them. 
```

Author: You are totally right, you can specify as many items (panes) as you want. They will just go one after another divided by splitter bars.

The following base options are available for a pane:

- [resizable](/Documentation/ApiReference/UI_Components/dxSplitter/Configuration/items/#resizable) (default: `true`)    
If true, a handle appears at the side of the pane. Drag the handle to adjust the pane size.

- [collapsible](/Documentation/ApiReference/UI_Components/dxSplitter/Configuration/items/#collapsible) (default: `false`)    
If true, an arrow appears on the handle. Click the arrow to close the pane.

- [collapsed](/Documentation/ApiReference/UI_Components/dxSplitter/Configuration/items/#collapsed) (default: `false`)    
Specifies whether the pane is initially collapsed.

> To configure the layout of the Splitter component, you can specify its [orientation](/Documentation/ApiReference/UI_Components/dxSplitter/Configuration/#orientation) as *'vertical'* or *'horizontal'* (default). All Splitter "size" properties depend on the orientation. If it is *'horizontal'*, the "size" means width. Otherwise, it is height. The following "size" properties can be specified:

Reviewer 2: ```suggestion
To configure the layout of the Splitter component, you can specify its [orientation](/Documentation/ApiReference/UI_Components/dxSplitter/Configuration/#orientation) as *'vertical'* or *'horizontal'* (default). All Splitter "size" properties depend on the orientation. If it is *'horizontal'*, then "size" means width. Otherwise, it is height. The following "size" properties can be specified:
```

> To configure the layout of the Splitter component, you can specify its [orientation](/Documentation/ApiReference/UI_Components/dxSplitter/Configuration/#orientation) as *'vertical'* or *'horizontal'* (default). All Splitter "size" properties depend on the orientation. If it is *'horizontal'*, then "size" means width. Otherwise, it is height. The following "size" properties can be specified:

Reviewer: ```suggestion
To configure the layout of the Splitter component, you can specify its [orientation](/Documentation/ApiReference/UI_Components/dxSplitter/Configuration/#orientation) as *'vertical'* or *'horizontal'* (default). All Splitter "size" properties depend on the orientation. If it is *'horizontal'*, "size" is the width. Otherwise, it is height. The following "size" properties can be specified:
```

- [size](/Documentation/ApiReference/UI_Components/dxSplitter/Configuration/items/#size)    
Initial pane width/height.

- [minSize](/Documentation/ApiReference/UI_Components/dxSplitter/Configuration/items/#minSize)    

> The minimum width/height to which a pane can be resized.

Reviewer 2: ```suggestion
The minimum width/height for a resizable pane.
```

- [maxSize](/Documentation/ApiReference/UI_Components/dxSplitter/Configuration/items/#maxSize)    

> The maximum width/height to which a pane can be resized.

Reviewer 2: ```suggestion
The maximum width/height for a resizable pane.
```

> You can also place one Splitter inside another to create advanced layouts. For this, use the [splitter](/Documentation/ApiReference/UI_Components/dxSplitter/Configuration/items/#splitter) property.

Reviewer 2: Those layouts are not necessarily "advanced". It's just one splitter inside another. They can be fairly simple. 

```suggestion
You can also place one Splitter inside another to create nested layouts. For this, use the [splitter](/Documentation/ApiReference/UI_Components/dxSplitter/Configuration/items/#splitter) property.
```

> Each Splitter pane can contain various content, from simple HTML markup to components. You can declare HTML markup inside the item tag or use the following properties to populate panes with content:

Reviewer 2: The phrase "various content" doesn't sound right to me. 

```suggestion
Splitter panes can contain content of different types, from simple HTML markup to components. You can declare HTML markup inside the item tag or use the following properties to populate panes with content:
```
